client/web/src/enterprise/batches/create/RunServerSideModal.module.scss

```
@@ -0,0 +1,65 @@
+.modal {
+    width: 55rem;
+    height: 25rem;
+}
+
+.content {
+    display: flex;
+    height: 95%;
+    justify-content: space-between;
+}
+
+.close {
+    display: flex;
+    position: absolute;
+    padding: 0;
+    top: 1rem;
+    right: 1rem;
+    border: none;
+    background: transparent;
+}
+
+.icon {
+    height: 1.125rem;
+    width: 1.125rem;
+    color: var(--icon-color);
+}
+
+.left {
+    display: flex;
+    flex-direction: column;
+    width: 72%;
+}
+
+.heading {
+    height: 15%;
+    font-size: small;
+}
+
+.video-container {
+    height: 85%;
+    background-color: var(--body-bg);
+}
+
+.right {
+    width: 25%;
```

Rather than rely on a percentage for the vertical spacing between right and left, which we're implicitly doing here by having 72% on `.left` and 25% on `.right` = 3% in between, I might suggest specifying a margin in rem to be consistent with the way we define spacing in other places, and then tell this container to occupy the remaining space with `flex: 1`. This way just helps us maintain predictable, consistent spacing (e.g. always a multiple of 0.5rem) that scales consistently as the text on the screen scales.

```suggestion
    margin-left: 1.5rem;
    flex: 1;
```